projects.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<head>
+<link href="https://fonts.googleapis.com/css?family=Josefin+Sans:100&display=swap" rel="stylesheet">
+</head>
+
+<header>
+  <nav>
+    <h1 class="name">Nicole Tanner</h1>
+    <li class="nav"><a href="index.html">Home</a></li>
+    <li class="nav"><a href="about.hmtl">About Me</a></li>
+    <li class="nav"><a href="projects.html">Projects</a></li>
+    <li class="nav"><a href="education.html">Education</a></li>
+    <li class="nav"><a href="#contact">Contact</a></li>
+   <nav>
+</header>
+    
+<main id="home">
+  <h1 class="welcome"> Projects</h1>
+  <h2 class="welcome_subtitle">Feel free to shift through and see what you like!</h2>
+ </main>
+    
+ <div id="projects_section" class= "projects_section">
+  
+  <h2 class="project_opener"> Here are some sites I've created...</h2>
+  
+  <h2 class="project_title"> To-Do List</h2>
+  <img class="projectpic" src="https://images.unsplash.com/photo-1547480053-7d174f67b557?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1650&q=80" alt="Check list on a desk">
+       
+  <h2 class="project_title">Cooky Cookie's Recipe Card</h2>
+  <img class="projectpic" src="https://images.unsplash.com/photo-1512058454905-6b841e7ad132?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1586&q=80" alt="Recipe on counter">
+    
+  <h2 class="project_title"> Tabata Timer</h2>
+  <img class="projectpic" src="https://images.unsplash.com/photo-1431499012454-31a9601150c9?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1650&q=80" alt="Timer in hand">
```

Comment:
Instead of fetching the images from a external URL. Download them and reference them locally in a `/image` folder. Make sure to optimize the images to be 2x the dimension they will appear on the page. https://imageoptim.com/mac